src/Sulu/Component/Util/ArrayUtils.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*
+ * This file is part of the Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Util;
+
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+
+/**
+ * Collection utilities working with symfony-expressions.
+ */
+final class ArrayUtils
```

Comment:
Would you really make that class final? And it would be nice to have this merged with `SortUtils`, but that's probably a BC break...